bibliotheca.py

```diff
@@ -47,7 +47,7 @@
 from util.http import HTTP
 from util.xmlparser import XMLParser
 
-class ThreeMAPI(object):
+class BibliothecaAPI(object):
```

Comment:
I decided since I was already changing all the signatures, the time was right to rename all remaining "3M" stuff to "Bibliotheca".